src/main.cpp

```diff
@@ -4867,6 +4873,157 @@ void static ProcessGetData(CNode* pfrom)
     }
 }
 
+bool checkVerificationQueue(){
+    return verificationQueue.isEmpty();
+}
+
+void txVerificationThread(){
+    int i = 0;
+    while(!ShutdownRequested())
+    {
+        if (!checkVerificationQueue() && i < 100)
```

Comment:
The value is hardcoded, it would be better to have a #define or a static const int defined somewhere. Moreover, could it be useful to have it set via an options in the zend?